aircan/dags/api_ckan_import_to_bq.py

```diff
@@ -0,0 +1,58 @@
+
+import logging
+import time
+import json
+import ast
+from datetime import date, datetime
+
+# Local imports
+from aircan.dependencies.google_cloud.bigquery_handler import bq_import_csv
+
+# Third-party library imports
+from airflow import DAG
+from airflow.exceptions import AirflowException
+
+from airflow.models import Variable
+from airflow.operators.python_operator import PythonOperator
+from airflow.utils.dates import days_ago
+
+
+args = {
+    'start_date': days_ago(0),
+    'params': { 
+        "path": "path/to/my.csv", 
+        "format": "CSV",
+        "ckan_resource_id": "res-id-123",
+        "schema": "['field1', 'field2']", 
+        "ckan_api_key": "API_KEY",
+        "ckan_site_url": "URL",
+        "output_bucket": str(date.today()),
+        "bq_project_id": "bq-project",
```

Comment:
AFter using the suggestions on https://github.com/datopian/aircan/pull/71, you may also want to organize the bq params into something like a nested json, 

```json
"resource": {},
"ckan_config": {},
"big_query": {}
```